README.md

```diff
@@ -1,23 +1,54 @@
 # clp-bench
-clp-bench is a tool for benchmarking [CLP] as well as other log management tools. The tool itself is
-a Python package, and we also provide a [web interface][ui] for viewing results.
 
-The methodology for the benchmarks is described [here](docs/methodology.md).
+**clp-bench** is a tool for benchmarking [CLP] and other log management systems. It functions as a 
+Python package and includes a [web interface][ui] for displaying benchmark results.
 
 ## Requirements
 
-* Docker
-* Python v3.10 or higher
+- Docker
+- Python v3.10 or higher
 
-# Set up
+# Setup
 
 ```shell
 python3 -m venv venv
 . venv/bin/activate
 pip install -e .
 ```
 
-You can use `clp-bench --help` to see usage instructions.
+To view usage instructions, run `clp-bench --help`.
 
+# Results
+You can view the current benchmark results [here][CLPBench UI]. The benchmark currently evaluates
+ingestion and query performance for the following tools:
+
+| Tool Name                      | Version     |
+|--------------------------------|-------------|
+| [ClickHouse][ClickHouse]       | 23.3.1.2823 |
+| [CLP][CLP]                     | 0.2.0       |
```

Comment:
```suggestion
| [clp][glt]                     | 0.2.0       |
```

Let's change the link to [GLT](https://docs.yscope.com/clp/main/user-guide/core-unstructured/glt.html).

Reply:
Is there a naming convention for the reference? (e.g., for ClickHouse shall I change it to clickHouse etc.?)

Reply:
will fix all comments by tonight